maven/maven.bzl

```diff
@@ -297,10 +343,17 @@ def maven_repository_specification(
         # The name of the repository
         name,
 
-        # The dictionary of artifact:sha256 entries used to populate this repository
+        # The dictionary of artifact -> properties which allows us to specify artifacts with more details.  These
+        # properties don't include the group, artifact name, version, classifier, or type, which are all specified
+        # by the artifact key itself.
+        #
+        # The currently supported properties are:
```

Comment:
Some mention that these are exclusive? Also, didn't notice -- do you blow up if both are set?

Reply:
Done and done.